VirtualPetAdoptionCenter.Web/Pages/MyPets.cshtml

```diff
@@ -0,0 +1,108 @@
+﻿@page
+@using VirtualPetAdoptionCenter.Core.Services;
+@using VirtualPetAdoptionCenter.Models.DomainModels;
+@model VirtualPetAdoptionCenter.Web.Pages.MyPetsModel
+
+<label class="title-label"> My pets </label>
+
+<head>
+    <link rel="stylesheet" href="~/css/PetStyle.css" />
+</head>
+
+<nav style="height: 75px; padding: 25px 50px; margin-top: 49px; margin-bottom: 50px;">
+    <ul style="display: inline-block;">
+        <li><a href="/AllPets">Home</a></li>
+        <li><a href="/MyPets">MyPets</a></li>
+        <li><a href="/Account">Account</a></li>    
+    </ul>
+    @if (Model.Achievement != null && Model.Achievement.Feed == 1)
+    {
+        <img src="https://cdn-icons-png.flaticon.com/512/5863/5863510.png" alt="Feed Achievement" title="Feed Achievement" style=" width: 50px; height: 50px; margin-top: -10px; margin-right: 15px; float: right;">
+    }
+    @if (Model.Achievement != null && Model.Achievement.Adopt == 1)
+    {
+        <img src="https://cdn-icons-png.flaticon.com/512/6410/6410115.png" alt="Adopt Achievement" title="Adopt Achievement" style=" width: 50px; height: 50px; margin-top: -10px; margin-right: 15px; float: right;">
+    }
+    @if (Model.Achievement != null && Model.Achievement.Environment == 1)
+    {
+        <img src="~/image/istockphoto-1199032156-612x612.jpg (1).png" alt="Environment Achievement" title="Environment Achievement" style=" width: 50px; height: 50px; margin-top: -10px; margin-right: 15px; float: right;">
+    }
+</nav>
+
+
+<section class="articles">
+    @for (var i = 0; i < Model.Pets.Count(); i++)
+    {
+        var pet = Model.Pets[i];
+        var condition = Model.PetConditions[i];
+
+        <article>
+            <div class="article-wrapper">
+                <figure>
+                    @if (pet.EnvironmentType == "Garden")
```

Comment:
You could create a helper method or partial view to encapsulate this logic and avoid repetition.